basics-assignment-1-start/basics-assignment-1-start/src/app/conrol-dictate/conrol-dictate.component.html

```diff
@@ -0,0 +1,27 @@
+
+<button (click)="changeAlerts()">Change Alerts</button>
+
+<mat-card *ngIf="selectedAlert" class="custom-card">
+  <mat-card-content>
+    <app-success-alert></app-success-alert>
+  </mat-card-content>
+</mat-card>
+
+<mat-card *ngIf="!selectedAlert" class="custom-card">
+  <mat-card-content>
+    <app-warning-alert></app-warning-alert>
+  </mat-card-content>
+</mat-card>
+
+<label for="repeatCount">Repeat Count:</label>
+<mat-select id="repeatCount" [(ngModel)]="repeatCount">
+    <mat-option *ngFor="let count of [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]" [value]="count">{{ count }}</mat-option>
```

Comment:
the array should not be part of the template. 
it should be a readonly field in the class component.